src/uniprotkb/adapters/uniProtkbConverter.ts

```diff
@@ -123,11 +126,14 @@ const uniProtKbConverter = (data: UniProtkbAPIModel): UniProtkbUIModel => {
     [EntrySection.NamesAndTaxonomy]: convertNamesAndTaxonomy(dataCopy),
     [EntrySection.SubCellularLocation]: convertSubcellularLocation(dataCopy),
     [EntrySection.DiseaseAndDrugs]: convertDiseaseAndDrugs(dataCopy),
+    [EntrySection.Phenotypes]: convertDiseaseAndDrugs(dataCopy),
```

Comment:
The changes to this file are slightly disconcerting to me. It generates duplicated logic, and make some logic run multiple times with the same output (3 times `convertDiseaseAndDrugs`).